tools/rpm.cc

```diff
@@ -6,6 +6,7 @@
 #include <rpm/rpmps.h>
 #include <rpm/rpmstring.h>
 #include <rpm/rpmts.h>
+#include <rpm/rpmfileutil.h>
```

Comment:
Why do we needs this here. With all the new functions going into rpmmacro_internal.hh ? Or should this be in some later patch?

Reply:
Oh, that's just a debugging leftover. Thanks for spotting, I'll drop.

Reply:
Dropped.